src/sagas.js

```diff
@@ -237,7 +237,8 @@ function* doCheckTokenExpired (action) {
       // No matter saga is canceled for whatever reason, the reload must happen, so here comes the ugly setTimeout()
       setTimeout(() => {
         console.info('======= doCheckTokenExpired() issuing page reload')
-        window.top.location.reload(true)
+        // window.top.location.reload(true)
+        window.location.href = `${AppConfiguration.applicationURL}/`
```

Comment:
Thanks for pointing that.

On dev environment (means with webpack's dev server), this will work neither.

The oVirt SSO is bypassed and sso token is injected into the application before it's start.
For that reason, "page refresh" leads to repeatedly use of the old token.
To fix that, we would need to design login in `DEV` environment differently - to follow same flow as in production. Not an easy task. Since it's for development only, I think it's low-priority.

Anyway, I removed the extra slash `/` so it at least looks better now.

Re-verified on production environment, still works.